src/core/SablierNFTDescriptor.sol

```diff
@@ -52,7 +52,7 @@ contract SablierV2NFTDescriptor is ISablierV2NFTDescriptor {
         bool isTransferable;
         string json;
         bytes returnData;
-        ISablierV2Lockup sablier;
+        ISablierLockup sablier;
```

Comment:
Should it be renamed as `lockup`?

Reply:
no specific preferences, both work IMO

if we decide to rename it, we should for all:

```solidity
ISablierLockup sablier;
string sablierModel;
string sablierStringified;
```

Reply:
Forgot to add a point that the choice of `sablier` vs `lockup` depends on the fact that whether the same NFT descriptor would be used for Lockup as well as Flow. If both, then `sablier` would be the right choice and if not, then `lockup` would be the right choice.

Reply:
https://github.com/sablier-labs/v2-core/pull/994/commits/513843469ebc4e746045fad2712f975b452965ff